Makefile

```diff
@@ -6,7 +6,9 @@ GOOS=linux
 # Ignore errors if there are no images.
 CONTROL_PLANE_IMAGES=./control-plane/cmd/kafka-controller ./control-plane/cmd/webhook-kafka ./control-plane/cmd/post-install
 TEST_IMAGES=$(shell find ./test/test_images ./vendor/knative.dev/reconciler-test/cmd ./vendor/knative.dev/eventing/test/test_images -mindepth 1 -maxdepth 1 -type d 2> /dev/null)
-KO_DOCKER_REPO=${DOCKER_REPO_OVERRIDE}
+TEST_IMAGE_TAG=latest
+DOCKER_REPO_OVERRIDE=
+KO_FLAGS=
```

Comment:
can we do something along like `KO_FLAGS="${KO_FLAGS:-}"` ?

Reply:
Why?  This isn't intended as an environment variable but as a make parameter.

Reply:
@matzew it's not necessary. It can be empty. It's in a makefile.